src/tree/tenants/TenantResourceTreeDataProvider.ts

```diff
@@ -43,36 +43,39 @@ export class TenantResourceTreeDataProvider extends ResourceTreeDataProviderBase
     }
 
     async onGetChildren(element?: ResourceGroupsItem | undefined): Promise<ResourceGroupsItem[] | null | undefined> {
-        if (element) {
-            return await element.getChildren();
-        } else {
-            const subscriptionProvider = await getAzureSubscriptionProvider(this);
-            const children: ResourceGroupsItem[] = await OnGetChildrenBase(subscriptionProvider);
+        return await callWithTelemetryAndErrorHandling('tenantResourceTreeDataProvider.onGetChildren', async (context: IActionContext) => {
```

Comment:
Nathan might have input here, but do we want to expose our implementation details in telemetry?

A better name might be `azureTenantsView.getChildren`?